src/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [2.0.0] - 2020-06-08
```

Comment:
That's not how unreleased works. This version should not be here until there is actually something that is being tagged. For the time being, everything here would be "unreleased".

Reply:
fixed